qemu/tests/transfer_file_over_ipv6.py

```diff
@@ -72,14 +75,28 @@ def get_file_md5sum(file_name, session, timeout):
 
     # config ipv6 address host and guest.
     host_ifname = params.get("netdst")
+    host_address = utils_net.get_host_ip_address(
```

Comment:
could you explain why need to add the param "linklocal"?

Reply:
@xiywang Updated.